src/devices/Hx711/HX711.cs

```diff
@@ -0,0 +1,274 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Device;
+using System.Device.Gpio;
+using System.Linq;
+using UnitsNet;
+
+namespace Iot.Device.Hx711
+{
+    /// <summary>
+    /// Hx711 - Weight scale Module
+    /// </summary>
+    /// <remarks>DataSheet: https://html.alldatasheet.com/html-pdf/1132222/AVIA/Hx711/109/1/Hx711.html</remarks>

```

Comment:
links to spec should only be in the README.md

Reply:
Ok, i remove it.